operations/loyaltymemberships.md

```diff
@@ -175,16 +167,16 @@ Adds loyalty memberships to the enterprise. Note this operation supports [Portfo
 | `AccessToken` | string | required | Access token of the client application. |
 | `Client` | string | required | Name and version of the client application. |
 | `ChainId` | string | optional | Unique identifier of the chain. Required when using [Portfolio Access Tokens](../concepts/multi-property.md), ignored otherwise. |
-| `LoyaltyMemberships` | array of [Loyalty membership add parameters](loyaltymemberships.md#loyalty-membership-add-parameters) | required, max 1000 items | Loyalty memberships to be added. |
+| `LoyaltyMemberships` | array of [Loyalty membership parameters](loyaltymemberships.md#loyalty-membership-parameters) | required, max 1000 items | Loyalty memberships to be added. |
 
-#### Loyalty membership add parameters
+#### Loyalty membership parameters
 
 | Property | Type | Contract | Description |
 | :-- | :-- | :-- | :-- |
 | `AccountId` | string | required | Unique identifier of the customer account. |
 | `LoyaltyProgramId` | string | required | Unique identifier of the loyalty program. |
 | `IsPrimary` | boolean | required | Defines the primary loyalty membership for the account. |
-| `State` | [Loyalty membership state](loyaltymemberships.md#loyalty-membership-state) | required | State of the loyalty membership. |
+| `State` | [Loyalty membership state](loyaltymemberships.md#loyalty-membership-state) | optional | State of the loyalty membership. |
```

Comment:
issue: Changing this to optional is the right call, but you should mention what happens when you omit the value.